webapp/api/github.py

```diff
@@ -239,7 +240,7 @@ def check_permissions_over_repo(
         response = self._request(
             "GET",
             f"repos/{owner}/{repo}/collaborators/{username}/permission",
-            raise_exceptions=False,
+            raise_exceptions=True,
```

Comment:
:+1:

Reply:
Ah yeah, thought it was probably good to raise here for future